test/inductor/test_compiled_optimizers.py

```diff
@@ -312,6 +330,82 @@ def test_fn(self):
     return test_fn
 
 
+class CompiledOptimizerParityTests(TestCase):
+    @optims(optim_db, dtypes=[torch.float32])
+    def test_correctness(self, device, dtype, optim_info):
```

Comment:
You could just add a decorator here...this infra was built to be simple:

```
@onlyCUDA
@optims...
def test_correctness....
```

Reply:
Some should run on CPU though right? that will only run all of them on cuda